### PR TITLE
Reformulated parts of the post.

### PR DESCRIPTION
**Pull request by matejak**
I have suggested a couple of modifications to the text without changing the meaning.

### CODE REVIEW

**Comment by jan-cerny** [comment]
LGTM, @Honny1 what do you think?

**Comment by Honny1** [comment]
I think it is great.:)

File: _posts/2020-8-24-visualising-the-results-of-complex-security-rules-using-oval-graph-tool.md [hunk 1]
```diff
@@ -96,7 +97,12 @@ This command generates a graph and saves a file named ```graph-of-<rule_id>-<dat
 
 You can read more about generating ARF report files using OpenSCAP in the OpenSCAP User [Manual](https://github.com/OpenSCAP/openscap/blob/maint-1.3/docs/manual/manual.adoc). Or you can use test arf files from [repository](https://github.com/OpenSCAP/OVAL-visualization-as-graph) /tests/test_data.
 
-Here is a live demo of the generated file. This graph is interactive. You can hide part of the rule or display any information about the test in the rule. In the graph we can see in the blink of an eye where the problem is.
+Here is a live demo of the generated file. This graph is interactive. You can
```
Comment (Honny1) [inline _posts/2020-8-24-visualising-the-results-of-complex-security-rules-using-oval-graph-tool.md:100]:
```suggestion
Here is a live demo of the generated file. This graph is interactive. You can: 
```

File: _posts/2020-8-24-visualising-the-results-of-complex-security-rules-using-oval-graph-tool.md [hunk 2]
```diff
@@ -96,7 +97,12 @@ This command generates a graph and saves a file named ```graph-of-<rule_id>-<dat
...
+
+* hide part of the rule criteria tree, or
```
Comment (Honny1) [inline _posts/2020-8-24-visualising-the-results-of-complex-security-rules-using-oval-graph-tool.md:102]:
```suggestion
* hide part of the rule criteria tree
```

File: _posts/2020-8-24-visualising-the-results-of-complex-security-rules-using-oval-graph-tool.md [hunk 3]
```diff
@@ -96,7 +97,12 @@ This command generates a graph and saves a file named ```graph-of-<rule_id>-<dat
...
+* display any information about the test in the rule.
```
Comment (Honny1) [inline _posts/2020-8-24-visualising-the-results-of-complex-security-rules-using-oval-graph-tool.md:103]:
```suggestion
* display any information about the test in the rule
```

**Comment by matejak** [comment]
The bulleted list differs from a Czech one.
UK English: http://www.plainenglish.co.uk/punctuating-bulleted-lists.html
US English: https://www.grammarly.com/blog/bullet-points

So it looks like that putting a comma at the end of a point is redundant, and so is the period at the end. Or one could use semicolons with a period, but that's the weird UK stuff.
The leading colon is redundant though.

**Comment by Honny1** [comment]
You are right! @matejak

**Pull request merged**